Homewrok4

DariaZhdanova-dev opened the pull request:
@Cakemanator

DariaZhdanova-dev committed:
First task

first task with additional part

DariaZhdanova-dev committed:
Shared directory for students creation

DariaZhdanova-dev committed:
Task 3

Tests inside

DariaZhdanova-dev committed:
update task2

Cakemanator commented on task1:
```diff
@@ -0,0 +1,38 @@
+
+## main part with users manipulation 
+#!/bin/bash
+
```
Can't see changes regarding "All new accounts should change their passwords every 30 days."

DariaZhdanova-dev replied:
Добавила изменения в task1 ( task1+30, изменения файла /etc/login.defs)
Не совсем поняла, как вычленить именно новые аккаунты группы, поэтому сделала цикл по всем юзерам группы, надеюсь, не совсем не то

Cakemanator replied:
Речь шла именно про `/etc/login.defs` )

DariaZhdanova-dev committed:
task 1 + 30 days

Добавила изменения в файл /etc/login.defs, изменив дефолтное значение PASS_MAX_DAYS на 30 и отдельно поменяла это значение для Боба
